From d7bff914f6d1de659f6f12fe186e13f5828bee6d Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Fri, 20 Dec 2013 12:11:08 +0800
Subject: [PATCH 2916/5965] fix drivers/amlogic/input/sensor/Makefile

this will make compile error when you don't select sensor.
if don't need sensor, don't compile any file.
---
 drivers/amlogic/input/sensor/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)
 mode change 100644 => 100755 drivers/amlogic/input/sensor/Makefile

diff --git a/drivers/amlogic/input/sensor/Makefile b/drivers/amlogic/input/sensor/Makefile
old mode 100644
new mode 100755
index c60e468acc92..f2be7ade3ca0
--- a/drivers/amlogic/input/sensor/Makefile
+++ b/drivers/amlogic/input/sensor/Makefile
@@ -6,7 +6,8 @@
 
 #Gravity
 
-obj-y += common.o
+obj-$(CONFIG_SENSOR_DEVICES) += common.o
+
 obj-$(CONFIG_GRAVITY_MMA8452)   += mma8452.o
 obj-$(CONFIG_GRAVITY_MMA7660)   += mma7660.o
 obj-$(CONFIG_GRAVITY_LIS3DH)   += lis3dh_acc.o
-- 
2.19.0

